src/python/antares_xpansion/xpansionConfig.py

```diff
@@ -167,6 +167,7 @@ def _set_default_settings(self):
             "log_level": "0",
             "separation_parameter": "0.5",
             "batch_size": "0",
+            "problems_format": "SAVED",
```

Comment:
The default value is ok even for open source solvers, as they will replace .svf to .mps and write files anyway, am I correct ?

Reply:
yes